### DISCUSSION TITLE
Cant connect to Mysql Server

### DISCUSSION THREAD

**Discussion by t2vee** [NamelessMC/Nameless#1976] [Q&A]
**Describe the issue**
Whenever i attempt to connect to the mysql server via the online installer it gives a 
"1045 - Access denied for user 'aurealisforums'@'localhost' (using password: YES)"
error. I have read previous thread/issues to no avail. I have also tried resetting ubuntu's permissions and changing mysql's password multiple times. For simple fixes like a different browser i have tried with again no avail.

**To Reproduce**
1. Setup NamelessMC (v2.0.0 pre7) as normal on Ubuntu 16.04 
2. Use the browser installation and attempt to connect to a Mysql server

**Expected behavior**
To receive a error as shown 

**Screenshots**
![image](https://user-images.githubusercontent.com/68762600/102619977-6544c080-4191-11eb-8803-a7e7e0f9d37a.png)

**NamelessMC version (from AdminCP -> Overview)**
NamelessMC v2.0.0 pre7 
Running on Apache2 - Ubuntu 16.04
Hosted on Google Cloud Compute Platform E2-small Plan

I have looked everywhere. I greatly appreciate any help!

**Comment by tadhgboyle** [comment]
There is not too much we can do as this is not an issue with namelessmc.
 Nameless uses native mysql drivers to connect - similar ones to phpmyadmin or other tools. 
What I mean is: namelessmc doesn't decide if your credentials are correct, it only returns what the MySQL server says.
We have no way to know your correct credentials.

**Comment by samerton** [comment]
To add to Aberdeener's comment, things to check are the username/password combination are correct, the user has access to the database (for example you've granted all privileges to the database) and the user is allowed to connect from the IP address you've entered.

Try searching for the error message online, there might be steps people performed to give their user access.

**Comment by Derkades** [comment]
Your MySQL server might be using the wrong authentication plugin. See this: https://stackoverflow.com/a/46908573

Side note, you should upgrade your Ubuntu installation, it is very old at this point. It contains old versions of software and is nearing EOL for regular users.
